cpp/include/eidir_graph.hpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+namespace cugraph {
+template class GraphViewBase<int32_t, int32_t, float>;
+template class GraphViewBase<int32_t, int32_t, double>;
+template class GraphViewBase<int32_t, int64_t, float>;
+template class GraphViewBase<int32_t, int64_t, double>;
+template class GraphViewBase<int64_t, int32_t, float>;
```

Comment:
Same comment on <int64_t, int32_t, *> throughout this file.

Reply:
Those don't exist in that header anymore. I fixed that a few days ago.